contracts/tests/TestVolOracle.sol

```diff
@@ -0,0 +1,63 @@
+//SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.7.3;
+
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import {Welford} from "../libraries/Welford.sol";
+import {DSMath} from "../libraries/DSMath.sol";
+import {VolOracle} from "../core/VolOracle.sol";
+
+contract TestVolOracle is DSMath, VolOracle {
+    using SafeMath for uint256;
+    uint256 private _price;
+
+    constructor(
+        address _pool,
+        address _baseCurrency,
+        address _quoteCurrency,
+        uint32 _period
+    ) VolOracle(_pool, _baseCurrency, _quoteCurrency, _period) {}
+
+    function mockCommit() external {
```

Comment:
This function is a copy-paste from `VolOracle.commit` so we can test out our logic. Did this because it's hard to mock a Uniswap pool.